src/v1/integer.js

```diff
@@ -802,6 +819,24 @@ Integer.MAX_VALUE = Integer.fromBits(0xFFFFFFFF|0, 0x7FFFFFFF|0, false);
 Integer.MIN_VALUE = Integer.fromBits(0, 0x80000000|0, false);
 
 /**
+ * Minimum safe native value.
+ * @type {!Integer}
+ * @const
+ * @inner
+ * @private
+ */
+var MIN_NATIVE = Integer.fromValue(Number.MIN_SAFE_INTEGER);
+
+/**
+ * Maximum safe native value.
+ * @type {!Integer}
+ * @const
+ * @inner
+ * @private
+ */
+var MAX_NATIVE = Integer.fromValue(Number.MAX_SAFE_INTEGER);
+
```

Comment:
Is there a reason you have added these as variables in module scope? Perhaps you could attach them to the `Integer` constructor like so:
```
Integer.MIN_NATIVE = Integer.fromValue(Number.MIN_SAFE_INTEGER)
Integer.MAX_NATIVE = Integer.fromValue(Number.MAX_SAFE_INTEGER)
```

I'd do it myself, but I'm not sure if mucking about with someone's PR is frowned upon :)